integration_test/third_party_apps_data/applications/tomcat/centos_rhel/install

```diff
@@ -1,39 +1,27 @@
 set -e
 
 sudo yum install -y curl java
-sudo mkdir -p /opt/tomcat/stage
-sudo curl "https://dlcdn.apache.org/tomcat/tomcat-9/v9.0.58/bin/apache-tomcat-9.0.58.tar.gz" -o /opt/tomcat/stage/tomcat.tgz
-sudo tar -xvzf /opt/tomcat/stage/tomcat.tgz -C /opt/tomcat --strip 1
-
-cat << EOF > tomcat.service
+sudo yum install tomcat -y
+sudo cat << EOF | sudo tee /usr/lib/systemd/system/tomcat.service
```

Comment:
Does the tomcat package not come with a systemd unit? Either way you shouldn't be overwriting anything in `/usr/lib`. If you need to add something to the unit file, you should be writing an override file in `/etc/systemd/system` (and there should be a good reason to do that).

Reply:
Updated and there needs to be updated to the service file to enable jmx.

Reply:
Thanks!